Pidaq/Lib/uploadToDrive/driveClass.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import print_function
+
+import logging
+import os
+import json
+from datetime import datetime
+from googleapiclient.discovery import build
+from httplib2 import Http
+from oauth2client import file, client, tools
+
+class uploadCsv:
+    def __init__(self):      
+        self.SCOPES = 'https://www.googleapis.com/auth/drive'
+        self.store = file.Storage('credentials.json') 
+        self.creds = self.store.get()
+
+        if not self.creds or self.creds.invalid:
+            self.flow = client.flow_from_clientsecrets('client_secret.json', self.SCOPES)
+            self.creds = tools.run_flow(self.flow, self.store)
+            
+        self.DRIVE = build('drive', 'v3', http= self.creds.authorize(Http()))
+
+        # Change this for the ID of the default folder 
+        self.defaultID = '1VDOOjuOeyNRJgdKmmRTJC1HVjq91F4Q8'
+            
+        # Searches for the Json file containing folder names and IDS
+        # These IDs are to be stored manually.
+        try:
+            with open('drive_options.json') as json_file:
+                self.parentDictionary = json.load(json_file)
+                
+        except FileNotFoundError:
+            logging.error('Creating a drive options dictionary file...')
+            
+            newDictionary = {'default': self.defaultID}
+            with open('drive_options.json', 'w') as new_file:    
+                json.dump(newDictionary, new_file)
+    
+            with open('drive_options.json') as json_file:
+                self.parentDictionary = json.load(json_file)
+
+
+    def execute(self, filePath, parentName):
+        """ Uploads a csv file from a local directory to a folder on Google Drive.
+        Parameters:
+        "filePath" is the path of the csv file itself and
+                    should be passed without quotations.
+        "parentName" is the name of the testing folder in the drive
+                and only a specific set of values are allowed.
+        The parent folder is supposedly going to be fixed for a
+        long period of time.
+        """
+                # Searches for the parent name in the dictionary
+        lowerDictionary = {}
+        for key, value in self.parentDictionary.items():
+            lowerDictionary[key.lower()] = value
+        try:
+            self.parentFolderID = lowerDictionary[parentName.lower()]
+        except NameError:
+            # Uploads to a default folder if the folder name does
+            # Not exist in the dictionary
+            self.parentFolderID = self.parentDictionary['default']
+            logging.error('Folder name not found \nuploading to "Default"')

```

Comment:
Again, maybe this could be a logging.warning, @colton-smith you probably have a better feel for this than me though, thoughts?